rust/src/jsonbuilder.rs

```diff
@@ -1061,12 +1061,16 @@ mod test {
         assert_eq!(jb.buf, r#"{"first":true"#);
         jb.set_bool("second", false).unwrap();
         assert_eq!(jb.buf, r#"{"first":true,"second":false"#);
+	jb.close().unwrap();
+	let _value: serde_json::Value = serde_json::from_str(&jb.buf).unwrap();
```

Comment:
Could this be done at a more unique point only if some rust `cfg` is there ?

Side note : we should try to fuzz and eve-validator the eve.json output

Reply:
> Could this be done at a more unique point only if some rust `cfg` is there ?

For what reason?  `serde_json` is already marked as a dev only dependency, so it being linked is already confined to unit tests.

Reply:
To validate all json output, not only some tests ?

Reply:
Why another `cfg` though? This validation would still just be in test scope right?

Reply:
No, I meant to run it for every S-V test, not just for unit tests...
(but maybe this is a bad idea)

Reply:
> No, I meant to run it for every S-V test, not just for unit tests... (but maybe this is a bad idea)

S-V test are already subject to JSON validation, so its out of scope here I think.

Reply:
You said

> Introduce serde_json as a dev dependency to validate our generated JSON.

So, if S-V test are already subject to JSON validation, why do we need more validation ?

Reply:
S-V is not a replacement for unit tests.  We have code that builds JSON which should be unit-tested and validated as well.